src/containers/createEvent/ducks/actions.ts

```diff
@@ -0,0 +1,16 @@
+import { genericAsyncActions } from '../../../utils/asyncRequest';
+import { NewEventInformation } from './types';
+
+export const create = genericAsyncActions<NewEventInformation, any>();
+export const edit = genericAsyncActions<NewEventInformation, any>();
+export const deleteEvent = genericAsyncActions<number, any>();
+
+export type CreateEventActions =
```

Comment:
_Technically_ they're not only actions to create events...

But I can't think of a name that encompasses all three without using the word "edit" 😅.

If you can think of a more generalized name feel free to change it, otherwise it's fine.

Reply:
EventControlActions is all we could come up with 😬